spark-k8s/versions.py

```diff
@@ -1,37 +1,19 @@
 versions = [
-    {
-        "product": "3.5.2",
-        "java-base": "17",
-        "java-devel": "17",
-        "python": "3.11",
-        "hadoop": "3.3.6",  # Hadoop version defined in ../hbase/versions.py to reduce build time and disk requirements
-        "hbase": "2.6.1",  # current Stackable LTS version
-        "aws_java_sdk_bundle": "1.12.367",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws/3.3.6
-        "azure_storage": "7.0.1",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-azure/3.3.6
-        "azure_keyvault_core": "1.0.0",  # https://mvnrepository.com/artifact/com.microsoft.azure/azure-storage/7.0.1
-        "jackson_dataformat_xml": "2.15.2",  # https://mvnrepository.com/artifact/org.apache.spark/spark-core_2.13/3.5.1
-        "stax2_api": "4.2.1",  # https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-xml/2.15.2
-        "woodstox_core": "6.5.1",  # https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-xml/2.15.2
-        "vector": "0.43.1",
-        "jmx_exporter": "1.1.0",
-        "tini": "0.19.0",
-        "hbase_connector": "1.0.1",
-    },
     {
         "product": "3.5.5",
         "java-base": "17",
         "java-devel": "17",
         "python": "3.11",
-        "hadoop": "3.3.6",  # Hadoop version defined in ../hbase/versions.py to reduce build time and disk requirements
-        "hbase": "2.6.1",  # current Stackable LTS version
-        "aws_java_sdk_bundle": "1.12.367",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws/3.3.6
-        "azure_storage": "7.0.1",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-azure/3.3.6
+        "hadoop": "3.4.1",  # Hadoop version defined in ../hbase/versions.py to reduce build time and disk requirements
```

Comment:
Oh nice, I didn't realise we could bump it for an existing version.
Hopefully we can do the same for Hive (otherwise I have to keep 3.3.6 around).

Reply:
I'll add a comment in there because it's misleading. Spark is not built against this version at all. This argument is only used to extract the hadoop/cloud connectors from the Hadoop LTS version and add them to the Spark classpath.

Reply:
https://github.com/stackabletech/docker-images/pull/1094/commits/36c90241d2becbb5a5cf09e35cd956bfec8135b4